ee_plugin/processing/export_geotiff.py

```diff
@@ -22,11 +22,17 @@ class ExportGeoTIFFAlgorithm(QgsProcessingAlgorithm):
     OUTPUT = "OUTPUT"
 
     def initAlgorithm(self, config: dict) -> None:
-        raster_layers = [
-            layer.name()
-            for layer in Map.get_iface().mapCanvas().layers()
-            if layer.providerType() == "EE"
-        ]
+        try:
+            raster_layers = [
+                layer.name()
+                for layer in Map.get_iface().mapCanvas().layers()
+                if layer.providerType() == "EE"
+            ]
+        except Exception as e:
+            raster_layers = []
+            local_context.log(f"Failed to retrieve raster layers: {e}")
+        if not raster_layers:
+            raster_layers = ["<No EE layers found>"]
```

Comment:
I'm not sure if a fake entry in this `list` is an appropriate way to handle this.  I think an empty list and perhaps a way to surface the error to the user would be more appropriate.